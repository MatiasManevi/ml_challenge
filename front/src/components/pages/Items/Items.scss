@import 'assets/scss/variables/spacing';

.items__link {
	text-decoration: none;
}
