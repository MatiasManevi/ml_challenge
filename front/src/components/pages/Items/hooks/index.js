export { default as useItems } from 'components/pages/Items/hooks/useItems';
