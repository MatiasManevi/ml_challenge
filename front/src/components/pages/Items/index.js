export { default } from 'components/pages/Items/Items';
