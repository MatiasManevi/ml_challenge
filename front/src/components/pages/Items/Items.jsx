import React from 'react';
import { Link } from 'react-router-dom';

import './Items.scss';
import { Item, BreadcrumbMenu } from 'components/organisms';
import { Content } from 'components/templates';
import { useItems } from './hooks';

const Items = () => {
	const { items } = useItems();

	return (
		<Content>
			<div className="items">
				<BreadcrumbMenu />
				{items.map((item) => (
					<Link key={item.id} className="items__link" to={`/items/${item.id}`}>
						<Item item={item} />
					</Link>
				))}
			</div>
		</Content>
	);
};

export default Items;
