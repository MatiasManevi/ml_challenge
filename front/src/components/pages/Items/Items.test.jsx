import React from 'react';
import { shallow } from 'enzyme';

import Items from './Items';

describe('Items', () => {
	it('should match snapshot', () => {
		const wrapper = shallow(<Items />);
		expect(wrapper).toMatchSnapshot();
	});
});
