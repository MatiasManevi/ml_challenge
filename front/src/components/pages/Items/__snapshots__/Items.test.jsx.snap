// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Items should match snapshot 1`] = `
<Content>
  <div
    className="items"
  >
    <BreadcrumbMenu />
    <Link
      className="items__link"
      to="/items/1"
    >
      <Item />
    </Link>
    <Link
      className="items__link"
      to="/items/2"
    >
      <Item />
    </Link>
  </div>
</Content>
`;
